packages/mui-material/src/ListItemButton/ListItemButton.js

```diff
@@ -177,7 +177,8 @@ const ListItemButton = React.forwardRef(function ListItemButton(inProps, ref) {
       <ListItemButtonRoot
         ref={handleRef}
         href={other.href || other.to}
-        component={(other.href || other.to) && component === 'div' ? 'a' : component}
+        // BaseButton processes href when component is set to 'button'
+        component={(other.href || other.to) && component === 'div' ? 'button' : component}
```

Comment:
https://github.com/mui/material-ui/blob/dfc0b89c665de8e5e3e4ecf68d8c782e74dbb65c/packages/mui-material/src/ButtonBase/ButtonBase.js#L293-L295

By looking at this, I think we can go with `button`. What do you think? @siriwatknp

Reply:
This is so confusing at first but I finally get it 😅.

I hope that in the future, there should be no inheritance.